hub/parameters/digitalhubdev.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "digitalhub_databaseAccount_name": {
```

Comment:
well we can build the name up by concatenating the parameter to an existing string which means yes that the parameters file should only include one value